test/e2e/specs/blocks/blocks__coblocks__blocks.ts

```diff
@@ -27,7 +26,11 @@ const features = envToFeatureKey( {
 	COBLOCKS_EDGE: envVariables.TEST_ON_ATOMIC || envVariables.COBLOCKS_EDGE,
 } );
 
-describe( DataHelper.createSuiteTitle( 'CoBlocks: Blocks' ), () => {
+/**
+ * This spec requires the following:
+ * 	- theme: a non-block-based theme (eg. Twenty-Twenty One)
+ */
+describe( 'CoBlocks: Blocks', function () {
```

Comment:
This is just part of the ongoing `DataHelper.createrSuiteTitle` removal, plus additional notes required for the spec to function.